libcontainer/system/sysconfig.go

```diff
@@ -10,3 +10,7 @@ import "C"
 func GetClockTicks() int {
 	return int(C.sysconf(C._SC_CLK_TCK))
 }
+
+func GetLongBit() int {
+	return int(C.sysconf(C._SC_LONG_BIT))
```

Comment:
This might be nicer:

```
/*
#include <limits.h>
#ifndef _SC_LONG_BIT
#define _SC_LONG_BIT (CHAR_BIT * sizeof(long))
#endif
*/
import "C"
```